zerver/views/invite.py

```diff
@@ -152,6 +165,9 @@ def invite_users_backend(
     ]
     check_role_based_permissions(invite_as, user_profile, require_admin=require_admin)
 
+    if welcome_bot_custom_message is not None and not user_profile.is_realm_admin:
```

Comment:
How does organization setting control sending custom message during invites? Should we allow it if organization setting is disabled? Although admins are only allowed to include custom messages and they can modify the organization setting but still.